pkg/blockstorage/azure/azuredisk.go

```diff
@@ -1,8 +1,5 @@
 // TODO: Switch to using the latest azure sdk and remove nolint.
 // Related Ticket- https://github.com/kanisterio/kanister/issues/1684
-//
-//nolint:staticcheck
-
```

Comment:
I believe this is the result of https://github.com/kanisterio/kanister/pull/2500/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9R2

Reply:
done